nulib/dc-api-v2
# Add /works/{id}/thumbnail route

To test:

1. Run Meadow in your dev environment
2. Create one or more works
3. Assign representative images
4. Make sure both the Indexer and the Reindexer complete a pass before you shut down Meadow
5. Start the API locally
6. Request /works/[work-id]/thumbnail for each of the works you created in step 2
7. Request thumbnails for works that don't exist to make sure they return 404
8. If you want to be extra thorough, create some corrupt work records in the work index with bad or missing thumbnail URLs and try to request them
